src/containers/StructurePage/StructureResources.jsx

```diff
@@ -30,25 +32,40 @@ export class StructureResources extends React.PureComponent {
     };
     this.getAllResourceTypes = this.getAllResourceTypes.bind(this);
     this.getTopicResources = this.getTopicResources.bind(this);
+    this.getArticle = this.getArticle.bind(this);
   }
 
   async componentDidMount() {
     try {
-      const { params: { topic1, topic2, topic3 } } = this.props;
+      const { params: { topic1, topic2, topic3 }, currentTopic } = this.props;
       await this.getAllResourceTypes();
       const topicId = topic3 || topic2 || topic1;
       this.getTopicResources(topicId);
+      if (currentTopic.contentUri) {
+        this.getArticle(currentTopic.contentUri);
+      }
     } catch (error) {
       console.log(error);
```

Comment:
Blir veldig mye støy på linting. Er det noe annet vi kan gjøre med dette? Foreslår å lage en funksjon som gjør dette, for akkurat nå støyer dette veldig mye.